### PR TITLE
Notification component

### PR DESCRIPTION
**Pull request by devbold**
### Notifications
-Design a Transaction Notifications Component for Header

**Refe:** 
screenshot 2018-10-11 15 12 22

**Branch:** notification-component

### CODE REVIEW

**Comment by Agusx1211** [comment]
It will be really nice to add a link to the tx page on Etherscan, there is no way of seeing the TX status of a pending/confirmed operation.

image
https://ropsten.etherscan.io/tx/0x0ac9dd78d59feb642e6c36e5ee84b4870f273eef6362046b53921c24e44cf647

File: src/app/core/header/notifications/notification-item/notification-item.component.ts [hunk 1]
```diff
@@ -10,8 +11,15 @@ export class NotificationItemComponent implements OnInit {
...
 
   progressbarMode = 'query';
+  urlAddress: String;
 
   constructor() {}
 
-  ngOnInit() {}
+  ngOnInit() {
+    if (environment.envName === 'dev') {
+      this.urlAddress = 'https://ropsten.etherscan.io/tx/';
```
Comment (Agusx1211) [inline src/app/core/header/notifications/notification-item/notification-item.component.ts]: We have this URL loaded on the network environment

https://github.com/ripio/rcn-loans/blob/e435122bf32f230b767d1f071354ee7126a80263/src/environments/environment.prod.ts#L32-L40

You can see an example on the lend button behavior

https://github.com/ripio/rcn-loans/blob/e435122bf32f230b767d1f071354ee7126a80263/src/app/shared/lend-button/lend-button.component.ts#L201

**Comment by Agusx1211** [comment]
There is a bug, the notification count reappears after realoding the screen.

ezgif com-video-to-gif

**Comment by devbold** [comment]
Approve reminder ✅ @Agusx1211

File: src/app/core/header/notifications/notifications.component.ts [hunk 2]
```diff
@@ -157,56 +157,25 @@ export class NotificationsComponent implements OnInit {
...
-    return Math.floor((new Date().getTime() - timestamp) / 1000);
-  }
-  getFormattedTime(timestamp: number) { // Receive a timestamp and formatted to a String
-    const delta: number = this.getTxTime(timestamp);
-    const deltaFormatted: string = Utils.formatDelta(delta);
-    if (delta <= 0) { return 'Just now'; }
-    return deltaFormatted;
-  }
-
-  // Update Tx Time
```
[the remark this answers is not in the document]
Comment (Agusx1211) [reply src/app/core/header/notifications/notifications.component.ts]: The functionality is still there, each time a Notification component is created the delta time is updated; also I disabled the display of seconds on the notification.

**Pull request merged**